distributed/shuffle/multi_comm.py

```diff
@@ -81,11 +81,11 @@ def __init__(
         self._exception: Exception | None = None
 
     @property
-    def queue(self):
+    def queue(self) -> asyncio.Queue:
         try:
             return MultiComm._queues[self._loop]
         except KeyError:
-            queue = asyncio.Queue()
+            queue: asyncio.Queue = asyncio.Queue()
```

Comment:
```suggestion
            queue: asyncio.Queue[None] = asyncio.Queue()
```

Since we only ever push `None` onto the queue AFAICT.